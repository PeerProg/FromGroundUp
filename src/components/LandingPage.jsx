import React, { useState, useEffect } from 'react';
import requestHandler from '../services/requestHandler'

const LandingPage = () => {
  const [landingPageUrl, setlandingPageUrl] = useState([]);

  useEffect(() => {
    requestHandler.homeUrl().then(res => setlandingPageUrl(res.data.message))
  }, [landingPageUrl])
  return (
    <div className="App">
      <div className="App-intro">
        <h2>
          {landingPageUrl}
        </h2>
      </div>
    </div>
  )
};

export default LandingPage;
