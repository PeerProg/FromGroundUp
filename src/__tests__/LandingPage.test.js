import React from 'react';
import LandingPage from '../components/LandingPage';
import renderer from 'react-test-renderer';

test('Landing Page Component', () => {
  const component = renderer.create(
    <LandingPage />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
