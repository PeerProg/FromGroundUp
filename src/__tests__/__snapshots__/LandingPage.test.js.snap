// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page Component 1`] = `
<div
  className="App"
>
  <div
    className="App-intro"
  >
    <h2>
      Landing Page Placeholder
    </h2>
  </div>
</div>
`;
